OutOfSchool/OutOfSchool.Common/Validators/CustomAgeAttribute.cs

```diff
@@ -96,4 +90,22 @@ private void EnsureThatMinAndMaxAgesAreValid()
             throw new InvalidOperationException("MaxAge cannot be less than MinAge.");
         }
     }
+
+    /// <summary>
+    /// Calculates min and max possible date.
+    /// </summary>
+    /// <param name="currentDate">Current date.</param>
+    /// <returns>Min and max possible dates.</returns>
+    private (DateOnly Min, DateOnly Max) CalculateMinAndMaxDate(DateOnly currentDate)
+    {
+        var maxYearOffset = currentDate.Year - 1;
+
+        var minAge = int.Clamp(MinAge, 0, maxYearOffset);
+        var maxAge = int.Clamp(MaxAge, 0, maxYearOffset);
```

Comment:
(Wow!) I had never expected to see person age limiting by 2+ thousand years. It's so unjustly to seniors )))